os/include/net/if.h

```diff
@@ -242,6 +242,37 @@ struct ifconf {
 #define ifc_buf	ifc_ifcu.ifcu_buf	/* Buffer address.  */
 #define ifc_req	ifc_ifcu.ifcu_req	/* Array of structures.  */
 
+#ifdef CONFIG_NET_NETMON
+/* This is Network monitor request */
+enum netmon_proto {
+	NETMON_INVALID = 0,
+	NETMON_TCP = 0x10,
+	NETMON_UDP = 0x20,
+	NETMON_UDPLITE = 0x21,
+	NETMON_UDPNOCHKSUM = 0x22,
+	NETMON_RAW = 0x40,
+	NETMON_TYPE_IPV6 = 0x08,
+};
+enum netmon_state {
+	NETMON_NONE,
+	NETMON_WRITE,
+	NETMON_LISTEN,
+	NETMON_CONNECT,
+	NETMON_CLOSE,
+};
+/* Socket info. */
+struct netmon_sock {
+	struct netmon_sock *flink;
+	enum netmon_proto type;
+	enum netmon_state state;
+	ip_addr_t local_ip;
+	ip_addr_t remote_ip;
+	u16_t local_port;
+	u16_t remote_port;
```

Comment:
there are uint16_t top of this file. I think using uint16_t would be better for consistency

Reply:
I got your point. Modification completed.